src/platform/Darwin/DnssdImpl.cpp

```diff
@@ -43,7 +43,7 @@ constexpr uint16_t kSRPTimeoutInMsec = 250;
 
 constexpr DNSServiceFlags kRegisterFlags        = kDNSServiceFlagsNoAutoRename;
 constexpr DNSServiceFlags kBrowseFlags          = kDNSServiceFlagsShareConnection;
```

Comment:
Do we need these constants kBrowseFlags, kGetAddrInfoFlags? could just use the kDNSServiceFlagsShareConnection in the api calls? we might want to keep them for future use cases where we might add to the flags

Reply:
I agree, I feel like seeing the flags inline at the call site would be more helpful than having these constants.

Reply:
It looks like `kRegisterFlags` is the only one of these constants that ends up being used more than once, and it's reused within a single function.

A middle ground would be to scope these constants to the functions where they are used rather than putting them in global scope, and that seems like the right solution for `kRegisterFlags` - the others are probably more clear with the flags inline.

Reply:
I would also add the flag for register and reconfirm records as well in the call site itself to be consistent since it doesn't have multiple flags there. If folks disagree I will revert that